src/App.svelte

```diff
@@ -1,84 +1,72 @@
 <script lang="ts">
-	 import AddCounter from './components/CounterAddButton.svelte'
-    import Counter from './components/Counter.svelte'
-    
-	function getNewCounter(){
-		return {name:"new",count:0,id:newid}
-	}
-	const newCounter = getNewCounter();
-	
-	let counters: {name:string,count:number,id:number}[] = [
-	{
-		name: 'new',
-		count: 0,
-		id: 0
-	}
-	];
-	$: list_titles = counters.map((counter)=>counter.name);
-	$: total = counters.length ? counters.map(item => item.count).reduce((prev, next) => prev + next) : 0;
-	$: newid = counters.length ? Math.max(...(counters.map((c)=>c.id))) + 1 : 0;
-;
+  import CounterAddButton from "./components/CounterAddButton.svelte";
+  import Counter from "./components/Counter.svelte";
 
-	const addCounter = (e:any) => {
-		
-		const newCounter = {
-			name: 'new',
-			count: 0,
-			id: newid
-		}
-		counters = [...counters, newCounter];
-	}
+  function getNewCounter() {
+    return { name: "new", count: 0, id: newid };
+  }
 
-	const removeCounter = (e:any) => {
-		counters = counters.filter((counter)=> counter.id !== e.detail)
-	}
+  let counters: { name: string; count: number; id: number }[] = [
+    getNewCounter(),
+  ];
+  $: listTitles = counters.map((counter) => counter.name);
+  let total: Number;
+
+  $: total = counters.reduce((total, counter) => total + counter.count, 0);
```

Comment:
前のPRで指摘しましたが、let totalは削除できるはずです

```suggestion
  $: total = counters.reduce((total, counter) => total + counter.count, 0);
```